src/run.js

```diff
@@ -226,8 +228,10 @@ const processPage = ({
         // First, go to the page with JavaScript disabled.
         await page.setJavaScriptEnabled(false)
         response = await page.goto(pageUrl)
-        if (!response.ok()) {
-          return safeReject(new Error(`${response.status()} on ${pageUrl}`))
+        if (response.status() >= 400) {
+          return safeReject(
+            new Error(`Response status ${response.status()} on ${pageUrl}`)
+          )
```

Comment:
Does this work? If someone does:
```sh
$ ./bin/minimalcss.js http://httpbin.org/redirect-to?url=https://www.peterbe.com
```
I definitely want it to allow the redirect to happen, but the next line `await page.content()` needs to be that of the page it redirected to.

Reply:
yes it does work without additional code, this part was causing troubles

```diff
       page.on('response', response => {
          const responseUrl = response.url()
          const ct = response.headers()['content-type'] || ''
-        if (!response.ok()) {
-          return safeReject(new Error(`${response.status()} on ${responseUrl}`))
-        }
```